shared_infra/lambdas/common/src/wellcome_lambda_utils/sns_utils.py

```diff
@@ -39,3 +39,11 @@ def publish_sns_message(sns_client, topic_arn, message):
     assert response['ResponseMetadata']['HTTPStatusCode'] == 200
 
     return response
+
+
+def extract_json_message(event):
+    """
+    Extracts a JSON message from an SNS event sent to a lambda
+    """
+    message = event['Records'][0]['Sns']['Message']
```

Comment:
Nice! But I see two places where you’re still repeating this line of code in this diff – could you push it out to other places please?

Reply:
Tha could go in sns_utils somehow as it's how we read events from sns in _every_ lambda that is triggered by sns. I think that should be a different consolidation patch though